composer.json

```diff
@@ -21,13 +21,13 @@
     "extra": {
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ^8.0",
```

Comment:
PHP does not follow [semantic versioning](https://semver.org).
```suggestion
        "php": "^7.3 || ~8.0.0",
```

Reply:
That's a decision for the laminas maintainers really. I've just replicated the existing style used by them on master.

Reply:
See the related issue report: #11 

> Modify composer.json to provide support for PHP 8.0 by adding the constraint `~8.0.0`

Reply:
Ah, sorry, yeah, missed that :)